### PR TITLE
bulid HomePage

### LINKED ISSUE #3: Adding Home page
adding a banner to the Home page

### PR DESCRIPTION
**Pull request by mariyasada**
adding a banner to the home page
adding epic deals and top brand's card to the home page by creating data (array of objects)
home
brand
.

### CODE REVIEW

File: src/components/HomePageCard/HomePageData.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const EpicDealItem = [
```
Comment (JantuDeb) [inline src/components/HomePageCard/HomePageData.js]: Use lower camel case for javascript variable

File: src/components/HomePageCard/HomePageCard.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import "../HomePageCard/HomePageCard.css";
+import { Link } from 'react-router-dom';
+
+
+ export const HomePageCard = ({Item}) => {
```
Comment (JantuDeb) [inline src/components/HomePageCard/HomePageCard.jsx]: Use user `item` instead of `Item`

**Review by JantuDeb — APPROVED** [review]
I have added some suggestions .

**Review by Raghav888 — APPROVED** [review]
Good naming convention and strcuture of code

File: src/components/HomePageCard/HomePageCard.jsx (hunk 2 above)
Comment (mariyasada) [reply src/components/HomePageCard/HomePageCard.jsx]: Ok. Done

**Pull request merged**